calendarext/build.gradle

```diff
@@ -37,6 +40,7 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation 'androidx.core:core-ktx:1.6.0'
+    coreLibraryDesugaring "com.android.tools:desugar_jdk_libs_nio:2.0.3"
```

Comment:
Is this really necessary?

Reply:
it will necessary if we use getTimeZone function that required java java 8. but we can remove it and change to TimeZone class instead.